packages/app/README.md

```diff
@@ -50,8 +50,11 @@ Then, open [localhost:8080](http://localhost:8080).
 Clone this repository, install dependencies and build the extension:
 
 ~~~bash
-git clone https://github.com/blockstack/ux && cd ux && yarn
-cd packages/app && yarn lerna run build --scope @stacks/connect-ui && yarn lerna run prod:ext
+git clone https://github.com/blockstack/ux
+cd ux
+yarn
+yarn run build:connect-ui
+yarn run build:extension
```

Comment:
`yarn run` is not needed - any script can be used. This is different than NPM btw, where you have to specify `run` for all non-standard scripts.

So, just `yarn build:connect-ui` and `yarn build:extension` work!